### PR TITLE
[MRG] DOC: Fix the space between attributes in docs

### PR DESCRIPTION
**Pull request by tirthasheshpatel**
I think sphinx doesn't allow a space between attributes and their description. I have tried to run a simple shell script to fix the issue.

### CODE REVIEW

**Review by rpgoldman — APPROVED** [review]
Thank you very much for doing this.  This has been on my to do list for ages.

**Comment by tirthasheshpatel** [comment]
You are welcome @rpgoldman :)

File: pymc3/tests/checks.py [hunk 1]
```diff
@@ -18,10 +18,10 @@
...
             np.abs(x - v) < bound,
-            x == v)), name + " out of bounds : " + repr(x) + ", " + repr(v) + ", " + repr(bound)
+            x == v)), name + " out of bounds: " + repr(x) + ", " + repr(v) + ", " + repr(bound)
 
 
 def close_to_logical(x, v, bound, name="value"):    
     assert np.all(np.logical_or(
             np.abs(np.bitwise_xor(x, v)) < bound,
-            x == v)), name + " out of bounds : " + repr(x) + ", " + repr(v) + ", " + repr(bound)
+            x == v)), name + " out of bounds: " + repr(x) + ", " + repr(v) + ", " + repr(bound)
```
Comment (ColCarroll) [inline pymc3/tests/checks.py:27]: looks unintentional here, but fine to me

File: pymc3/tests/test_variational_inference.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ def aevb_initial():
         (NormalizingFlowGroup, {'flow': 'radial'}),
         (NormalizingFlowGroup, {'flow': 'radial-loc'})
     ],
-    ids=lambda t: '{c} : {d}'.format(c=t[0].__name__, d=t[1])
+    ids=lambda t: '{c}: {d}'.format(c=t[0].__name__, d=t[1])
```
Comment (ColCarroll) [inline pymc3/tests/test_variational_inference.py:203]: looks unintentional here, but fine to me

File: pymc3/smc/smc.py [hunk 3]
```diff
@@ -307,7 +307,7 @@ def posterior_to_trace(self):
             size = 0
             for var in varnames:
                 shape, new_size = self.var_info[var]
-                value.append(self.posterior[i][size : size + new_size].reshape(shape))
+                value.append(self.posterior[i][size: size + new_size].reshape(shape))
```
Comment (ColCarroll) [inline pymc3/smc/smc.py:310]: looks unintentional here, but fine to me

File: pymc3/smc/smc.py [hunk 4]
```diff
@@ -436,7 +436,7 @@ def posterior_to_function(self, posterior):
         size = 0
         for var in self.variables:
             shape, new_size = var_info[var.name]
-            varvalues.append(posterior[size : size + new_size].reshape(shape))
+            varvalues.append(posterior[size: size + new_size].reshape(shape))
```
Comment (ColCarroll) [inline pymc3/smc/smc.py:439]: looks unintentional here, but fine to me

**Review by ColCarroll — APPROVED** [review]
very nice. couldn't believe a "simple shell script" wouldn't hit more false positives.

**Pull request merged**